### PR TITLE
Run command managed identities parameters for blobs

### PR DESCRIPTION
**Pull request by viveklingaiah**
### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ Yes] new API version (API version 2022-08-01 for Run Command PUT APIs, adds managedIdentities inputs for blobs used in RunCommand PUT APIs). The Runcommand v2 (PUT APIs) are in Preview at the moment (as far as communication with users). We are planning to take it to GA by Oct 31, 2022.
          - Managed identities is another way to provide user's VMs and VMSSs access to storage blobs that can contain user's script that is run on VM and VMSS using Run Command PUT APIs. Users have ability to provide SAS URIs of blobs today. Managed identities are more secure.
          - No changes to GET APIs
          - I have copied the RunCommand.json spec and examples from 2022-03-01 and made changes to PUT VM Run Command and PUT VMSS VM Run Command APIs and their examples.
          - Design Doc related to this change:
          https://microsoft.sharepoint.com/:w:/t/ComputeVM/EWTzlVJsh3hKpmDxD0KLi1oBFsVWwP2G7tmbXggij2XaXw?e=rlItUp           
      - [ Yes] update existing version for new feature
 
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.  - Oct 31 2022
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month. Oct 01, 2022 (After we take the Run Command 2022-08-01 APIs to GA)
 
  5. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.
  - SDKs need to be updated.

### Contribution checklist (MS Employees Only):
- [Yes ] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [ Yes] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ Yes] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review. 
> - Change to data plane APIs - No data plane Changes
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [Yes ] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new API version
    -[ Yes] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [Yes ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [Yes ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by vivlingaiah** [comment]
@ArcturusZhang ArcturusZhang, could you check and let me know if anything else needed from my side on this PR ? I have done my best to run the validation tools and fix.

File: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/runCommandExamples/VirtualMachineRunCommand_Get.json [hunk 1]
```diff
@@ -34,7 +34,9 @@
           "asyncExecution": false,
           "runAsUser": "user1",
           "timeoutInSeconds": 3600,
-          "provisioningState": "Succeeded"
+          "provisioningState": "Succeeded",
+          "outputBlobUri": "Base URI of a storage append blob(without SAS token)",
+          "errorBlobUri": "Base URI of a storage append blob(without SAS token)"
```
Comment (ArcturusZhang) [inline specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/runCommandExamples/VirtualMachineRunCommand_Get.json:39]: could you please put some "real values" in the example files? We should already have descriptions on these properties, we do not need to duplicate that in the examples.
The example files should be showing what these properties really look like in a real traffic

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]

**Review by Sandido — CHANGES_REQUESTED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/runCommand.json [hunk 2]
```diff
@@ -1163,14 +1177,18 @@
...
         "scriptUri": {
           "type": "string",
-          "description": "Specifies the script download location."
+          "description": "Specifies the script download location. It can be either SAS URI of an Azure storage blob with read access or public URI."
         },
         "commandId": {
           "type": "string",
           "description": "Specifies a commandId of predefined built-in script."
+        },
+        "scriptUriManagedIdentity": {
```
[the remark this answers is not in the document]
Comment (vivlingaiah) [reply specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/runCommand.json]: Introduced version 2022-11-01

File: specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/runCommandExamples/VirtualMachineRunCommand_Get.json (hunk 1 above)
Comment (vivlingaiah) [reply specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/runCommandExamples/VirtualMachineRunCommand_Get.json]: done

**Review by ArcturusZhang — COMMENTED** [review]
Please address the Avacado errors accordingly.
Also please add a new composite tag and a new only tag for your new api-version in readme.md

**Comment by grizzlytheodore** [comment]
PR looks good. I will proceed this with the rest of 2022-11-01 release when ready

**Review by grizzlytheodore — APPROVED** [review]

**Pull request closed without merging**